src/test/resources/tls_old/server-cert.pem

```diff
@@ -0,0 +1,17 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
@vietj yes it's garbage ! I made a mistake on committing this folder. I'll remove it.